lib/tasks/styleguide.rake

```diff
@@ -2,11 +2,7 @@ namespace :styleguide do
   desc "Generate Live Styleguide"
   task generate: :environment  do
     system "rspec spec/features/harvest_styleguide.rb"
-    system "ruby populate_styleguide.rb"
-  end
-
-  task watch: :environment do
-    system('nodemon --exec "ruby populate_styleguide.rb" -w app/views/styleguide_template.hbs populate_styleguide.rb -V')
```

Comment:
removed this rarely used :watch task in order to remove the `nodemon` dependency implicitly introduced [RW]